packages/flutter/test/material/date_range_picker_test.dart

```diff
@@ -250,12 +251,17 @@ void main() {
       await tester.pumpAndSettle();
     });
 
-    testWidgets('landscape', (WidgetTester tester) async {
+    testWidgetsWithLeakTracking('landscape', (WidgetTester tester) async {
       await showPicker(tester, kCommonScreenSizeLandscape);
       expect(tester.widget<Text>(find.text('Jan 15 – Jan 25, 2016')).style?.fontSize, 24);
       await tester.tap(find.text('Cancel'));
       await tester.pumpAndSettle();
-    });
+    },
+    // TODO(polina-c): remove after widgets/app.dart/defaultActions stops holding objects.
```

Comment:
defaultActions is a static that is supposed to live for the duration of the application. So its expected to never be GCed and therefore is not a leak.

Reply:
defaultActions holds some disposed objects from GC, that is leak. Some object on retaining path should release the references.

Reply:
What's the disposed object? From what I can tell, the objects in the defaultActions map are not disposable.

Reply:
The disposed object is `ValueNotifier<_OverlayEntryWidgetState?>`.
Full retaining path is looong and is linked to the issue: https://github.com/flutter/flutter/issues/130354

Reply:
This PR is just test covers three tests. 
I suggest to merge this PR and move the discussion to the issue.